### DISCUSSION TITLE
booster and uki

### DISCUSSION THREAD

**Discussion by soloturn** [anatol/booster#221] [Q&A]
i am reading [arch linux unifeid kernel image page](https://wiki.archlinux.org/title/Unified_kernel_image) and there is a slight difference how this is described for booster and dracut. booster is mentioned in the [ukify section](https://wiki.archlinux.org/title/Unified_kernel_image#ukify), while dracut is mentioned 2 times, one time that it [can produce an UKI on its own](https://wiki.archlinux.org/title/Dracut#Unified_kernel_image), or by using ukify. i do not see UKI mentioned in the bosster readme, so i wonder what are you recommending from a booster point of view, and why?

**Comment by ghost** [comment]
@soloturn I can't speak for anatol, but using booster as the initramfs within the EFI file generated from ukify should be fine if that's what you're asking. 

This assumes of course that ukify doesn't mutate the image during the generation process for some reason or by accident. On my system, I've actually been experimenting with this and haven't run into any issues with the EFI during boot but considering ukify is currently experimental prepare to have some fallback mechanism in-place.
